src/models/AdSlot.js

```diff
@@ -91,7 +91,9 @@ class AdSlot extends Base {
             description: this.description,
             fallbackUnit: this.fallbackUnit,
             minPerImpression: this.minPerImpression,
-            created: this.created,
+            created: this.created
```

Comment:
try `new Date(this.created || Date.now())`  or `this.created ? new Date(this.created) : new Date()`

Reply:
It will be best to add it in the market just to be sure, because it can came with wrong date from the platform (counting on user machine time). Adding it on the market will ensure the correct date.